core/sys/cc.h

```diff
@@ -110,6 +111,12 @@
 #define CC_INLINE
 #endif /* CC_CONF_INLINE */
 
+#ifdef CC_CONF_ALIGN_ATTR
+#define CC_ALIGN_ATTR CC_CONF_ALIGN_ATTR
+#else /* CC_CONF_INLINE */
+#define CC_ALIGN_ATTR
```

Comment:
@bthebaudeau 

I agree with you here. I've only kept it this way because that's how it was in cc26xx-rf.c and I wanted to avoid changing runtime behaviour for this PR. Note also that in the case of cc26xx-rf.c it really would be a bug to not align any of these variables.

Now that you've mentioned this specifically, however, I think that I will simply git rid of the fallback definition so that this generates an error. It really just doesn't make sense to do otherwise.